API_doc/Communication.md

```diff
@@ -0,0 +1,28 @@
+# Communication lib-service
+
+All orders must contain at least **ORDER**, containing the name of the order, and **CONFIG_ID**, containing the id of the config you want to work on (except for *CONFIG_CREATE*).
+Each response must contain at least **ORDER_STATE** (see below).
+
+### Orders
+| Order Name | Order Description | Additional Argument(s) | Additional Returned Value(s) |
+| -------- | -------- | -------- | -------- |
+|*CONFIG_CREATE*|Create a new config.|**CONFIG_NAME**|**CONFIG_ID**|
+|*CONFIG_LOAD*| Load an existing config. |*none*| **CONFIG_NAME**|
```

Comment:
Missing config id

Reply:
If you mean *config id* is missing in the arguments, I say on the top of the document that *config id* is always needed, so it's not written to avoid repetition

Reply:
Hum, I think there's a confusing: cnofig id != config. It's rather minor, but a config id is supposed to be used to load a config. Then you're supposed to use the real config (which will probably be associated with a separate identification for the protocol) to use it. The reason being: a config ID is potentially very long (this is on purpose, it increases security), while the protocol simply use a monotonously increasing value to refer to a config, which could then be stored in a long.
I also don't believe it's good style to say an argument is always there: it means that one can't read the document locally.

Reply:
Firstly, i want to say that i'm more for a solution that is close to JWT (json web token). And use this unique id to navigate through the database

Reply:
No. We aren't doing web, persistence isn't a problem, therefore JWT may not be applicable. Also you can keep the token on the server side if need be. Transmitting the long token rather than a simple 32 or 64 bit value has no practical gain.

Reply:
I think this problem is solved in the new version.